### ISSUE TITLE
j2objc 2.1.1 "-Xuse-jdt" problem

### ISSUE THREAD

**Issue by ivan-tsaryov** [google/j2objc#988]
Hello. I had been using j2objc-2.0.2 with translate flag "-Xuse-jdt" and it works fine. But after upgrading j2objc to 2.1.1 version a cannot use flag "-Xuse-jdt" anymore. I have removed it and got j2objcTranslate errors in one of my libraries, like: "cannot find symbol
  import javax.lang.model.type.DeclaredType;" and similar ones. How to fix this?

**Comment by tomball** [comment]
javax.lang.model.type.DeclaredType is part of Java 7, and j2objc requires that a Java 8 JDK be installed on your Mac. Run "/usr/libexec/java_home -V" to verify a Java 8 JDK is installed.

The other possible issue is if you are trying to translate Java source files that import DeclaredType. This class is not part of j2objc's [JRE emulation library](https://developers.google.com/j2objc/javadoc/jre/reference/javax/lang/model/type/package-summary), since it's used for modeling Java code, which doesn't make sense on iOS (that's the same reason why it's not part of Android). 

The -Xuse-jdt flag was removed once we finished switching j2objc's front-end from the Eclipse JDT to instead use javac. We had to move to javac since the JDT had several Java 8-related outstanding bugs -- javac isn't bug-free either, but it leads language releases instead of following them.
